v0/util/index.js

```diff
@@ -1112,6 +1112,14 @@ const isOAuthSupported = (destination, destHandler) => {
   );
 };
 
+function isAppleFamily(platform) {
+  return (
+    platform.toLowerCase() === "ios" ||
+    platform.toLowerCase() === "watchos" ||
+    platform.toLowerCase() === "ipados" ||
+    platform.toLowerCase() === "tvos"
+  );
```

Comment:
Helps to easily extend in the future.
```suggestion
  const appleOSNames = ["ios", "watchos", "ipados", "tvos"];
  return appleOSNames.includes(platform.toLowerCase());
```

Reply:
Resolved

Reply:
Refactored